tasks/pvwa_clean.yml

```diff
@@ -1,2 +1,46 @@
 ---
 # tasks file for pvwa clean
+
+- name: Clean Windows\\Temp Folder
+  win_find:
+    paths: "C:\\Windows\\Temp"
+    patterns: ['PVWA*.*', 'InstallationAutomation*.*']
+  register: files_to_delete
+
+- win_file:
+    path: "{{item.path}}"
+    state: absent
+  with_items: "{{ files_to_delete.files }}"
+
+- name: Clean cred files
+  win_file:
+    path: "{{item}}"
+    state: absent
+  with_items:
+    - "{{ pvwa_installation_path }}\\CredFiles\\appuser.ini"
+    - "{{ pvwa_installation_path }}\\CredFiles\\gwuser.ini"
+
+- name: clean run history
+  win_regedit:
+    path: 'HKCU:\\Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\RunMRU'
+    state: absent
+    delete_key: yes
+
+- name: clean event logs
+- win_shell: |
```

Comment:
Try to use this module:
https://docs.ansible.com/ansible/2.4/win_eventlog_module.html
@avishayil @yogevh

Reply:
It's better to not use it, as there is no task to clear all event viewer logs.